service/awsconfig/v5/resource/cloudformation/adapter/outputs.go

```diff
@@ -1,20 +1,70 @@
 package adapter
 
-import "github.com/giantswarm/aws-operator/service/awsconfig/v5/cloudconfig"
+import (
+	"strconv"
+
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/aws-operator/service/awsconfig/v5/cloudconfig"
+	"github.com/giantswarm/aws-operator/service/awsconfig/v5/key"
+)
 
 // The template related to this adapter can be found in the following import.
 //
 //     github.com/giantswarm/aws-operator/service/awsconfig/v5/templates/cloudformation/guest/outputs.go
 //
 
 type outputsAdapter struct {
-	MasterCloudConfigVersion string
-	WorkerCloudConfigVersion string
+	Master        outputsAdapterMaster
+	Worker        outputsAdapterWorker
+	VersionBundle outputsAdapterVersionBundle
+}
```

Comment:
The outputs adapter now obtains all information necessary to render the outputs template.